src/sidebar/components/sidebar-panel.js

```diff
@@ -9,12 +9,26 @@ import Button from './button';
 import Slider from './slider';
 import SvgIcon from '../../shared/components/svg-icon';
 
+/**
+ * @typedef SidebarPanelProps
+ * @prop {Object} [children]
+ * @prop {string} [icon] - An optional icon name for display next to the panel's title
+ * @prop {string} panelName
+ *    - A string identifying this panel. Only one `panelName` may be active at any time.
+ *    Multiple panels with the same `panelName` would be "in sync", opening and closing together.
```

Comment:
```suggestion
 * @prop {string} panelName -
 *    A string identifying this panel. Only one `panelName` may be active at any time.
 *    Multiple panels with the same `panelName` would be "in sync", opening and closing together.
```

Small convention nit - when prop docs are moved to multiple lines the ` - ` should still go after the name of the prop. Come to think of it, we could just omit it entirely in this case, assuming it doesn't cause any errors for TS's parsing, as the parameter name is already visually separated from the comments by a new line.

Reply:
If its okay with you, I'll leave it for now, but move it to the correct place. I like the separator.